components/src/core/components/Input/Autocomplete/AutocompleteInput.vue

```diff
@@ -149,7 +149,7 @@ export default defineComponent({
 
   computed: {
     showChips(): boolean {
-      return Array.isArray(this.modelValue) && this.multiple;
+      return Array.isArray(this.modelValue || []) && this.multiple;
```

Comment:
Why is this needed. With the new change Array.isArray will be true always because [] is an array.

Reply:
This component supposed to have arrays in modelValue when the multiple is true but modelValue gets null when clearing the field just after entering a text without selecting from the suggested dropdown.
therefore the outcome of this only getting false when the modelValue is a string or an int